Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Chapel/bibles.yml

```
@@ -1,14 +1,14 @@
 - type: entity
   parent: Bible
   id: BibleMystagogue
-  name: book of mysteries
-  description: The mystagogue's holy book.
+  name: field guide to advanced anomalous sciences
```

Books with actual titles should use TitleCase
```suggestion
  name: Field Guide to Advanced Anomalous Sciences
```